crates/sui-open-rpc/spec/openrpc.json

```diff
@@ -160,11 +160,11 @@
           }
         },
         {
-          "name": "flag",
+          "name": "sig_scheme",
           "description": "Flag of the signature scheme that is used.",
```

Comment:
Would it be possible to specify that eddsa is 0x00? (If it isn't already)

Reply:
I can add that in the description